### PR TITLE
Remove scanning connections log messages

### PR DESCRIPTION
**Pull request by JamesNK**
Fixes https://github.com/aspnet/SignalR/issues/2155

### CODE REVIEW

**Comment by davidfowl** [comment]
What is up with this spanish OS thing.

File: src/Microsoft.AspNetCore.Http.Connections/Internal/HttpConnectionManager.Log.cs [hunk 1]
```diff
@@ -25,15 +25,9 @@ private static class Log
             private static readonly Action<ILogger, string, Exception> _connectionTimedOut =
                 LoggerMessage.Define<string>(LogLevel.Trace, new EventId(5, "ConnectionTimedOut"), "Connection {TransportConnectionId} timed out.");
 
-            private static readonly Action<ILogger, Exception> _scanningConnections =
```
Comment (davidfowl) [inline src/Microsoft.AspNetCore.Http.Connections/Internal/HttpConnectionManager.Log.cs]: Should we leave a comment in saying what these were?

Comment (JamesNK) [reply src/Microsoft.AspNetCore.Http.Connections/Internal/HttpConnectionManager.Log.cs]: I like that. Can do

File: src/Microsoft.AspNetCore.Http.Connections/Internal/HttpConnectionsEventSource.cs [hunk 2]
```diff
@@ -84,17 +75,5 @@ public void ConnectionTimedOut(string connectionId)
                 }
             }
         }
-
-        [Event(eventId: 4, Level = EventLevel.Verbose, Message = "Scanning connections.")]
-        public void ScanningConnections()
```
Comment (anurse) [inline src/Microsoft.AspNetCore.Http.Connections/Internal/HttpConnectionsEventSource.cs]: Put a comment here as well.

**Pull request merged**